server/src/main/java/com/indexdata/reservoir/server/Main.java

```diff
@@ -0,0 +1,35 @@
+package com.indexdata.reservoir.server;
+
+import io.vertx.core.Vertx;
+import org.apache.logging.log4j.LogManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Main {
+  /**
+   * Main entry point for the Reservoir server.
+   *
+   * @param args command line arguments
+   */
+
+  public static void main(String[] args) {
+    System.out.println("Starting Reservoir server (standard output message)");
+
+    final org.apache.logging.log4j.Logger log2 = LogManager.getLogger(Main.class);
+
+    final Logger log = LoggerFactory.getLogger(Main.class);
+
+    log.info("Starting Reservoir server (SLF4J message)");
+    log2.info("Starting Reservoir server (Log4j2 message)");
+
+    Vertx.vertx()
+        .deployVerticle(new MainVerticle())
```

Comment:
When MainVerticle is started in the native binary there doesn't seem to be any metadata for the module name and version so it looks like this:
```
13:57:35 INFO  MainVerticle         Starting null null 7fa14d0126a4a0743005250d450665b3db715888
```

Reply:
fixed